scripts/cangen/can_generator.py

```diff
@@ -37,7 +37,7 @@ def _assert_valid_dbc(filename: str):
 
 
 def _parse_dbc_files(dbc_files: List[str]) -> Database:
-    logger.info(f"Parsing DBC files: {dbc_files}")
+    logger.debug(f"Parsing DBC files: {dbc_files}")
```

Comment:
I'd like this to stay "info" since it is a one time message and contains fairly important information